_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -36205,6 +36205,10 @@
 /turf/open/floor/wood,
 /area/maintenance/department/engine)
 "fms" = (
+/obj/machinery/door/poddoor/shutters/preopen{
+	id = "chemistry_shutters";
+	name = "chemistry shutters"
```

Comment:
I'd suggest you use capital letters on the names (so it'd be Chemistry Shutters instead of chemistry shutters), it's more consistent.